go/bundle/decoder.go

```diff
@@ -521,7 +540,7 @@ func loadMetadata(bs []byte) (*meta, error) {
 
 	r := bytes.NewBuffer(bs)
 
-	// Step 2. "If reading 10 bytes from stream returns an error or doesn't return the bytes with hex encoding "84 48 F0 9F 8C 90 F0 9F 93 A6" (the CBOR encoding of the 4-item array initial byte and 8-byte bytestring initial byte, followed by 🌐📦 in UTF-8), return a "format error"." [spec text]
+	// Step 2. "If reading 10 bytes from stream returns an error or doesn't return the bytes with hex encoding "86 48 F0 9F 8C 90 F0 9F 93 A6" (the CBOR encoding of the 6-item array initial byte and 8-byte bytestring initial byte, followed by 🌐📦 in UTF-8) or "85 48 F0 9F 8C 90 F0 9F 93 A6" (same as before, but a 5-item CBOR-array), return a "format error"." [spec text]
```

Comment:
These `[spec text]` comments are stale, as parsing algorithm is not part of the spec after #604.
We should clean up them (but not in this PR).